resources/views/register-expert-profile.blade.php

```diff
@@ -9,62 +9,79 @@
 
         <div class="con">
             <section>
-                <form action="" class="create-cso-form">
+                <form action="{{ route('store-expert') }}" class="create-cso-form" method="POST" enctype="multipart/form-data">
+                    @csrf
                     <div class="form-section">
                         <div class="flex">
-                            <div class="field">
-                                <label for="">Select user</label>
-                                <select name="organization-type" id="organization-type">
-                                    {{-- select options data from backend --}}
-                                    <option value="option">Option</option>
-                                </select>
-                            </div>
                             <div class="field">
                                 <label for="">Location</label>
                                 <input type="text" name="location" id="location" placeholder="Enter expert location"
-                                    value="" required>
+                                    value="{{ old('location') }}" required>
+                                    @error('location')
+                                        <span>{{$message}}</span>
+                                    @enderror
+                            </div>
+                            <div class="field">
+                                <label for="">Phone number</label>
+                                <input type="text" name="phone_number" id="phone_number" placeholder="+237656000000"
+                                    value="{{ old('phone_number') }}" required>
+                                    @error('phone_number')
+                                        <span>{{$message}}</span>
+                                    @enderror
                             </div>
                         </div>
                         <div class="flex">
```

Comment:
Remove status. Only the admin changes the status and this is also the case with cso registrations.